tag-release-candidate.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Why not reuse the same tag-release script? The only special thing about this one I can see is the release notes file.

Reply:
There would be a few more conditions. I wanted to keep them separate for now, we can merge later. In fact after we drop `latest_release`, these scripts should become nearly identical.